Description: Correct pod NAME issues i LedgerSMB/File/Part.pm
 Has the module name but not the dash plus short description in the NAME entry.
Bug: http://sourceforge.net/tracker/?func=detail&aid=3472597&group_id=175965&atid=1005191
Author: Robert James Clay <jame@rocasa.us>

--- a/LedgerSMB/File/Part.pm
+++ b/LedgerSMB/File/Part.pm
@@ -2,7 +2,7 @@
 
 =head1 NAME
 
-LedgerSMB::File::Part
+LedgerSMB::File::Part - Manages attachments to goods and services.
 
 =head1 SYNPSIS
 
